test/Faker/Provider/BaseTest.php

```diff
@@ -565,6 +566,31 @@ public function testRandomElements()
         $this->assertCount(3, $allowDuplicates);
         $this->assertContainsOnly('string', $allowDuplicates);
     }
+
+    /**
+     * @dataProvider testRegexifyDataProvider
+     *
+     * @param $expectedRegex
```

Comment:
@icanhazstring 

What do you think about adding types for the `@param` annotations?

Reply:
Sure, will add them